schemas/IDLs/org.gel.models.participant.avro/1.1.0-SNAPSHOT/CancerParticipant.avdl

```diff
@@ -69,7 +69,11 @@ protocol CancerParticipants {
         union {null, ProgrammePhase} programmePhase;
 
         /**
+<<<<<<< HEAD
+        The time when the sample was recieved. In the format YYYY-MM-DDTHH:MM:SS+0000
+=======
         The time when the sample was received. In the format YYYY-MM-DDTHH:MM:SS+0000
+>>>>>>> develop
```

Comment:
Looks like this has been commited from a conflict that wasn't resolved, this will need to be removed